external/IO/CMakeLists.txt

```
@@ -25,7 +25,7 @@ if (RADIUM_IO_ASSIMP)
         # For now, only configure in Debug mode to work with xcode
         ExternalProject_Add(assimp
             GIT_REPOSITORY https://github.com/assimp/assimp.git
-            GIT_TAG tags/v5.0.1
+            GIT_TAG master
             GIT_SHALLOW TRUE
```

```suggestion
            GIT_SHALLOW FALSE
```
in case we set a commit instead of a tag or branch name